generators/app/templates/src/app/components/Routes/index.ejs

```diff
@@ -1,21 +1,26 @@
-import React from 'react';
+import React, { Suspense, lazy } from 'react';
 import { ConnectedRouter } from 'connected-react-router';
 import { Switch } from 'react-router-dom';
 
 import { history } from '../../../redux/store';
-import Home from '../../screens/Dashboard';
+import Loading from '../Spinner/components/loading';
 
 import AuthenticatedRoute from './components/AuthenticatedRoute';
 import * as Routes from './constants';
 import styles from './styles.scss';
+
+const Home = lazy(() => import('../../screens/Dashboard'));
```

Comment:
Are we going to add a func like this for each screen that we add? I would think of a different approach

Reply:
This is how is done. This replaces the import, so it won't have more lines thatn before, just instead of doing `import Home  from ...` you have to do `const Home = lazy(() => import(...))`

Reply:
Sure, you are right... I didn't notice that the imports aren't here :D

Reply:
Sure, you are right... I didn't notice that the imports aren't here :D